src/core/sentry/index.ts

```diff
@@ -1,9 +1,24 @@
 import * as Sentry from '@sentry/browser'
+import { texts } from '../../i18n'
 import { env } from 'src/core/env'
 
 export function initSentry() {
   if (process.env.NODE_ENV === 'development') return
   Sentry.init({
     dsn: env.SENTRY_DSN,
+    beforeSend(event) {
+      if (event?.exception?.values?.some(((error) => error.value === 'user-feedback'))) {
```

Comment:
On peut rajouter un peu de contexte, car on détourne un peu le comportement

```suggestion
      const isUserFeedbackFromBanner = event?.exception?.values?.some(((error) => error.value === 'user-feedback'))

      if (isUserFeedbackFromBanner) {
```

Reply:
Effectivement, merci ! 😊